specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/examples/NetworkVirtualApplianceSiteDelete.json

```diff
@@ -7,8 +7,29 @@
     "networkVirtualApplianceName": "nva"
   },
   "responses": {
-    "200": {},
-    "202": {},
-    "204": {}
+    "200": {
+      "description": "Delete successful.",
+      "headers": {
+        "Location": {
+          "type": "string"
```

Comment:
This should be the value instead of schema.

Reply:
@raych1 I have updated the PR as per the review comment. please review and approve.

Reply:
Can you check the `ModelValidation` errors which have some more to fix?

Reply:
Fixed the requird modelvalidation issue. Thanks for your guidance.